### ISSUE TITLE
Elements with insufficient color contrast

### ISSUE THREAD

**Issue by andrewmarino** [unitymakesus/unitywebagency#2]
The following elements have insufficient contrast at this conformance level (Level AA).
**Success Criterion: 1.4.3: Contrast (Minimum)**

- [x] uabb-infobox-text-wrap — large bold text (expected 3:1, found 2.55:1)
- [x] uabb-infobox-text-wrap — regular text (expected 4.5:1, found 2.55:1)
- [x] uabb-infobox-title-prefix — regular text (expected 4.5:1, found 2.55:1)

<img width="690" alt="Screen Shot 2020-03-19 at 4 14 24 PM" src="https://user-images.githubusercontent.com/6686529/77110901-b81ffa80-69fc-11ea-91e6-a4e5f54ec371.png">

![Screen Shot 2020-03-19 at 3 39 22 PM](https://user-images.githubusercontent.com/6686529/77107981-d8998600-69f7-11ea-865e-65e216b2b281.png)
